src/st_widgets.c

```diff
@@ -645,6 +647,25 @@ static boolean WidgetEnabled(widgetstate_t state)
     return true;
 }
 
+static void ForceDoomFont(sbe_widget_t *widget)
+{
+    if (WidgetEnabled(hud_widget_font))
+    {
+        if (widget->font != hudfont)
+        {
+            widget->font = hudfont;
+        }
+    }
+    else
+    {
+        if (widget->font == hudfont && strcmp(widget->font_name, "Console"))
```

Comment:
Any way we get around multiple string comparisons per tic?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1992/commits/4cf091be1e45aa3473dfc779679a6e199359fc3f